app/config/keymaps.js

```diff
@@ -35,8 +36,8 @@ const _extend = function (customsKeys) {
   if (customsKeys) {
     for (const command in customsKeys) {
       if (command) {
-        commands[command] = customsKeys[command];
-        keys[customsKeys[command]] = command;
+        commands[command] = normalize(customsKeys[command]);
+        keys[normalize(customsKeys[command])] = command;
```

Comment:
Maybe better to have: `const normalizedKeys = normalize(customsKeys[command])` and use it in following assignations or use `keys[commands[command]] = command` directly.

But I think this file should be totally refactored. Several things should be improved. We'll work on that later.